src/views/studio/CreateEditChannelView/CreateEditChannelView.style.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from '@emotion/styled'
+import { sizes, breakpoints, transitions } from '@/shared/theme'
+import { Textarea, ActionBarTransaction, Avatar, FormField } from '@/shared/components'
+import { TitleSection } from '@/views/consumer/ChannelView/ChannelView.style'
+
+type AtionBarProps = {
+  isActive?: boolean
+}
+
+export const StyledTitleSection = styled(TitleSection)`
+  display: inline-flex;
+  width: auto;
+  flex-direction: row;
+  padding-top: ${sizes(8)};
+  @media (min-width: ${breakpoints.small}) {
+    padding-top: 0;
+  }
+`
+
+export const TitleContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+`
+
+export const InnerFormContainer = styled.div`
+  width: 100%;
+  max-width: var(--max-inner-width);
```

Comment:
Not sure if you would still need this one but this variable will still exists after #385 is merged